src/scripts/main.js

```diff
@@ -1 +1,24 @@
 'use strict';
+
+const gallery = document.querySelector('.gallery');
+
+// eslint-disable-next-line no-shadow
+gallery.addEventListener('click', (event) => {
+  if (event.target.closest('.list-item__link')) {
+    event.preventDefault();
+  }
+});
+
+const list = document.querySelector('ul');
+const imgForChange = document.getElementById('largeImg');
```

Comment:
Ensure that the element with the ID 'largeImg' exists in your HTML. If it doesn't, the `getElementById` method will return null, and calling `setAttribute` on it will cause an error.